x-pack/docs/en/ilm/update-lifecycle-policy.asciidoc

```diff
@@ -1,5 +1,432 @@
+          "rollover": {
+            "max_docs": 1
+          }
+        }
+      },
+      "delete": {
+        "minimum_age": "10d",
+        "actions": {
+          "delete": {}
+        }
+      }
+    }
+  }
+}
+------------------------
+// CONSOLE
+
+////
+[source,js]
+------------------------
+PUT my_index
+{
+  "settings": {
+    "index.lifecycle.name": "my_executing_policy"
+  }
+}
+------------------------
+// CONSOLE
+// TEST[continued]
+////
+
+The <<ilm-explain,Explain API>> is useful to introspect managed indices to see which phase definition they are currently executing.
+Using this API, we can find out that `my_index` is currently attempting to be rolled over.
+
+[source,js]
+--------------------------------------------------
+GET my_index/_ilm/explain
+--------------------------------------------------
+// CONSOLE
+// TEST[continued]
+
+[source,js]
+--------------------------------------------------
+{
+  "indices": {
+    "my_index": {
+      "index": "my_index",
+      "managed": true,
+      "policy": "my_executing_policy",
+      "skip": false,
+      "lifecycle_date": 1538475653281,
+      "phase": "hot",
+      "phase_time": 1538475653317,
+      "action": "rollover",
+      "action_time": 1538475653317,
+      "step": "attempt_rollover",
+      "step_time": 1538475653317,
+      "phase_execution": {
+        "policy": "my_executing_policy",
+        "modified_date_in_millis": 1538475653317,
+        "version": 1,
+        "phase_definition": {
+          "minimum_age": "0ms",
+          "actions": {
+            "rollover": {
+              "max_docs": 1
+            }
+          }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+// TESTRESPONSE[s/"lifecycle_date": 1538475653281/"lifecycle_date": $body.indices.my_index.lifecycle_date/]
+// TESTRESPONSE[s/"phase_time": 1538475653317/"phase_time": $body.indices.my_index.phase_time/]
+// TESTRESPONSE[s/"action_time": 1538475653317/"action_time": $body.indices.my_index.action_time/]
+// TESTRESPONSE[s/"step_time": 1538475653317/"step_time": $body.indices.my_index.step_time/]
+// TESTRESPONSE[s/"modified_date_in_millis": 1538475653317/"modified_date_in_millis": $body.indices.my_index.phase_execution.modified_date_in_millis/]
+
+Updating `my_executing_policy` to have no rollover action and, instead, go directly into a newly introduced `warm` phase.
+
+[source,js]
+------------------------
+PUT _ilm/my_executing_policy
+{
+  "policy": {
+    "phases": {
+      "warm": {
+        "minimum_age": "1d",
+        "actions": {
+          "forcemerge": {
+            "max_num_segments": 1
+          }
+        }
+      },
+      "delete": {
+        "minimum_age": "10d",
+        "actions": {
+          "delete": {}
+        }
+      }
+    }
+  }
+}
+------------------------
+// CONSOLE
+// TEST[continued]
+
+Now, version 2 of this policy has no `hot` phase, but if we run the Explain API again, we will see that nothing has changed.
```

Comment:
Maybe worth pointing out that the phase definition below still shows version 1 of the policy?

Reply:
added a comment ++